np.sh

```diff
@@ -15,7 +15,7 @@ if test "0" != "$(git rev-list --count --left-only @'{u}'...HEAD)"; then
 	exit 128;
 fi
 
-trashCli=$(node -e "var path = require('path');console.log(path.join(path.dirname(require('fs').realpathSync('$0')), 'node_modules/.bin/trash'))");
+trashCli=$(node -p "require.resolve('.bin/trash')");
```

Comment:
I'm not clear exactly what you mean. That's true, but it doesn't seem to affect how require.resolve works here. I've given this a quick test on my machine with NP installed either globally and locally inside a project, and it seems to work fine. Is there another case I'm missing?